presto-docs/src/main/sphinx/release/release-344.md

```diff
@@ -0,0 +1,64 @@
+# Release 344 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`murmur3` function. ({issue}`5054`)
+* Add {func}`from_unixtime_nanos` function. ({issue}`5046`)
+* Add [T-Digest](/functions/tdigest) type and functions. ({issue}`5158`)
+* Improve performance and latency of queries leveraging dynamic filters. ({issue}`5081`)
+* Allow to configure (using ``dynamic-filtering.service-thread-count`` feature toggle) number of
+  threads used for processing dynamic filters on coordinator. ({issue}`5341`)
+* Extend environment variables substitution (`${ENV:variable-name}`) to work also when the substituted text
+  is part of the configuration value. ({issue}`4345`)  
+* Remove `fast-inequality-joins` configuration property. The fast inequality joins remain enabled. ({issue}`5375`)
+* Report dynamic filters collection duration. ({issue}`5434`)  
+* Improve query performance by propagating more detailed dynamic filters to coordinator. ({issue}`5340`)  
+* Improve performance and accuracy of {func}`approx_percentile`. ({issue}`5158`)
+* Improve performance of cross join queries. ({issue}`5276`)
+* Prevent potential query deadlock when query runs out of memory.  ({issue}`5289`)
+* Fix failure due to rounding error when casting between two `timestamp` types with precision higher than 6. ({issue}`5310`)
+* Fix failure due to rounding error when casting between two `timestamp with time zone` types with precision higher than 3. ({issue}`5371`)
+* Fix column pruning for `EXPLAIN ANALYZE` ({issue} `4760`)
```

Comment:
Can share more details about what was the issue here? Do I understand this correctly (without reading #4760) that it is about enabling projection pushdown to connector for `EXPLAIN ANALYZE` which because of bug was not happening? 

Is "column pruning" common term for that?

Reply:
i think the gist here is that EXPLAIN ANALYZE was not correctly reflecting/analyzing actual query. Maybe we say that?

Reply:
What @findepi said is correct. The issue is that when running EXPLAIN ANALYZE the plan would scan all the columns even if the original query required only a subset.